backend/api/Services/MissionSchedulingService.cs

```diff
@@ -221,8 +221,20 @@ private async Task MoveInterruptedMissionsToQueue(IEnumerable<string> interrupte
                     continue;
                 }
 
-                missionRun.Status = MissionStatus.Pending;
-                await _missionRunService.Update(missionRun);
+                var newMissionRun = new MissionRun
```

Comment:
Could this change maybe be rebased into the first commit?

Reply:
I agree. However, the merge conflicts that arose were sufficiently difficult that I decided against it.